openhands/runtime/client/runtime.py

```diff
@@ -123,7 +123,7 @@ def __init__(
         self.action_semaphore = threading.Semaphore(1)  # Ensure one action at a time
 
         self.runtime_builder = DockerRuntimeBuilder(self.docker_client)
-        logger.debug(f'EventStreamRuntime `{sid}` config:\n{self.config}')
+        logger.debug(f'EventStreamRuntime `{sid}`')  # config:\n{self.config}')
```

Comment:
To be removed/reverted?

Reply:
I'll remove the commented out part. Even in my testing I don't find it useful anymore, but it spams the log.